docs/Examples.md

```diff
@@ -31,7 +31,7 @@ indexers. A `Standalone` resource can be used to create a single instance
 that can perform either, or both, of these roles.
 
 ```yaml
-apiVersion: enterprise.splunk.com/v1alpha2
+apiVersion: enterprise.splunk.com/v1beta1
```

Comment:
Gaurav, I am changing this as a part of this [PR](https://github.com/splunk/splunk-operator/pull/140). Should we keep all changes from v1alpha* to v1beta1 so that we are consistent?